src/Stripe.net/Services/Apps/Secrets/SecretCreateOptions.cs

```diff
@@ -1,10 +1,19 @@
 // File generated from our OpenAPI spec
 namespace Stripe.Apps
 {
+    using System;
```

Comment:
ooc why did these get added?

Reply:
`DateTime` is in the `System` namespace and `UnixDateTimeConverter` is in `Stripe.Infrastructure`.